scipy/spatial/distance.py

```diff
@@ -2740,7 +2753,20 @@ def cdist(XA, XB, metric='euclidean', *, out=None, **kwargs):
        proportion of those elements ``u[i]`` and ``v[i]`` that
        disagree where at least one of them is non-zero.
 
-    10. ``Y = cdist(XA, XB, 'chebyshev')``
+    10. ``Y = cdist(XA, XB, 'jensenshannon')``
+
+       Computes the Jensen-Shannon distance between two probability arrays.
+       Given two probability vectors, :math:`p` and :math:`q`, the
+       Jensen-Shannon distance is
+
+       .. math::
+
+          \\sqrt{\\frac{D(p \\parallel m) + D(q \\parallel m)}{2}}
+
+        where :math:`m` is the pointwise mean of :math:`p` and :math:`q`
+        and :math:`D` is the Kullback-Leibler divergence.
```

Comment:
Makes sense overall--note that there's currently a bit of strangeness in the rendered doc artifact I see in CI (some math rendering issue, and maybe the indentation is a little off for entries below 9):

![image](https://user-images.githubusercontent.com/7903078/118580115-88799900-b74c-11eb-862f-53627cce5fc0.png)

Reply:
Fixed rendering issue, and also fixed the indentation of all the other entries:
https://31178-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.spatial.distance.pdist.html
https://31178-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.spatial.distance.cdist.html